src/java/test/org/apache/zookeeper/ServerConfigTest.java

```diff
@@ -57,4 +59,16 @@ public void testTooManyArguments() {
         String[] args = {"2181", "/data/dir", "60000", "10000", "9999"};
         serverConfig.parse(args);
     }
-}
+
+    boolean checkEquality(String a, String b) {
+        return a != null && a.equals(b);
+    }
+
+    boolean checkEquality(String a, File b) {
+        try {
+            return a != null && b != null && new File(a).getCanonicalPath().equals(b.getCanonicalPath());
+        } catch (IOException e) {
```

Comment:
Oh, I see. It is because the method overloading. But could we add a comment like ``//DO NOTHING`` or maybe log the error in the ``catch`` body? Empty catch blocks make me uncomfortable. :) But I am fine as is, nevertheless.